nextcord/flags.py

```diff
@@ -486,6 +486,14 @@ def known_spammer(self):
         """
         return UserFlags.known_spammer.value
 
+    @flag_value
+    def active_developer(self):
+        """:class:`bool`: Returns ``True`` if the user is a Active Developer
```

Comment:
For consistency, add a full-stop.

```suggestion
        """:class:`bool`: Returns ``True`` if the user is a Active Developer.
```